src/home/carousel/styles/style-carousel.js

```diff
@@ -0,0 +1,19 @@
+import { makeStyles } from '@material-ui/core'
```

Comment:
if it`s under styles/ folder, we can drop the style- prefix.

Reply:
Done!